homeassistant/components/hue.py

```diff
@@ -192,7 +192,7 @@ def setup(self):
             self.bridge = phue.Bridge(
                 self.host,
                 config_file_path=self.hass.config.path(self.filename))
-        except ConnectionRefusedError:  # Wrong host was given
+        except (ConnectionRefusedError, OSError):  # Wrong host was given
```

Comment:
`ConnectionRefusedError` is a subclass of `OSError`.

Reply:
Oh.

Well good thing that I am going to throw it all away when we migrate to aiohue #12830

Reply:
I tested it locally before committing but only tested

```
>>> isinstance(ConnectionRefusedError, OSError)
False
```

But classes are of type Type.

Instantiating did the trick:

```
>>> isinstance(ConnectionRefusedError(), OSError)
True
```